check_os.py

```diff
@@ -117,3 +125,62 @@ def setup_logging(gui=False):
     logging.info_log_file = os.path.join(log_directory, info_filename)
     logging.error_log_file = os.path.join(log_directory, error_filename)
     logging.config.fileConfig(log_config, disable_existing_loggers=False)
+
+
+def check_leveldb(store_dir):
+    if not store_dir:
+        return
+    try:
+        from leveldb import LevelDB, LevelDBError
+        use_leveldb = True
+    except ImportError:
+        from Tribler.Core.plyveladapter import LevelDB
+        use_leveldb = False
+    except:
+        error_and_exit("Tribler Error", "LevelDB not found")
+
+    try:
+        db_path = store_dir.decode('windows-1252') if sys.platform == "win32" else store_dir
```

Comment:
Here we have quite some code duplication. We should extract this into a more generic method.

Reply:
You are right. I'll refactor the code.